test-d/automatic-s.tsx

```diff
@@ -41,6 +41,3 @@ expectError(<a invalid={[true]} />)
 // The automatic runtime the children prop to define JSX children, whereas it’s used as an attribute in the classic runtime.
 
 expectType<Result>(<a children={<b />} />)
-
-declare function Bar(props?: Record<string, unknown>): Element
-expectError(<Bar />)
```

Comment:
should probably be a bunch of tests here.
Also, could the JSX dev runtime be checked? 🤔

Reply:
I can try to add some tests. Is there anything in particular you're looking for? 

Also, I'm not quite sure how to check the JSX dev runtime (or even what it is)